scripts/prepare-release.ts

```diff
@@ -119,36 +119,35 @@ const incrementVersions = async (preid: string = 'beta') => {
       const modulePath = lernaModuleLocations[i];
       const json = JSON.parse(readFileSync(path.join(modulePath, 'package.json'), { encoding: 'utf-8' }));
       const tags = await getDistTags(json.name);
-      if (tags[preid]) {
-        const next = inc(tags[preid], 'prerelease', undefined, preid);
-        console.log(`Setting next version for ${json.name} to ${next}`);
-        json.version = next;
-        writeFileSync(
-          path.join(modulePath, 'package.json'),
-          JSON.stringify(json, null, 2) + '\n'
-        );
-        // since we're manually setting new versions, we must also reconcile all other lerna packages to now use the 'next' version for this module
-        lernaModuleLocations.forEach((otherModulePath) => {
-          // skip it for the current version
-          if (otherModulePath === modulePath) {
-            return;
+
+      const next = inc(tags[preid] || tags['latest'], 'prerelease', undefined, preid);
+      console.log(`Setting next version for ${json.name} to ${next}`);
```

Comment:
I can see that now some packages like web-demo, sdk-test etc which seems not published also are going through this logic flow. Probably the check was introduced to gate these internal packages probably. Lets wait for sdk teams feedback

Reply:
Web-demo and express should be ignored. They where deleted via github actions before this script is run.

Reply:
Lerna should ignore those set the aren't set to public.